joomla/joomla-cms
# [com_modules] positions view: client_id selector naming

#### Summary of Changes

The `client_id` seletor is named `client_id`, not `filter.client_id.`. This was uniformized in 3.6.0 across all modules, cache, templates and languages views (the ones that have the client_id selector).

One view remain: the positions view in com_modules. I think this view is unused in the core but still should be changed. So, this PR changes it.
#### Testing Instructions
- Apply patch
- Test `/administrator/index.php?option=com_modules&view=positions&layout=modal&tmpl=component&client_id=0`

(should show the site template positions)
- Test `/administrator/index.php?option=com_modules&view=positions&layout=modal&tmpl=component&client_id=1`

(should show the admin template positions)
- Test `/administrator/index.php?option=com_modules&view=positions&layout=modal&tmpl=component`

(should show the admin template positions, since it was the last one used)
